### DISCUSSION TITLE
FullPageOS for x86

### DISCUSSION THREAD

**Discussion by PPPDUD** [guysoft/FullPageOS#530] [Ideas]
I'm interested in using an x86 version of FullPageOS for a personal project. Can someone port this to x86 for me please?

**Comment by guysoft** [comment]
FullPageOS uses a base image that runs on RaspsberryPi, It can also be adapted to other single board computers.
In order for it to work on x86 you will need an base image for it to be ported to.
I would like to be optimistic, but since I am fully stretched out volunteering doing to 100% in my spare time. I doubt someone would just do it for free, it requires considerable work. You might be able to pay someone.
I am converting this to a discussion
